docroot/themes/custom/twentyseventeen/sass/components/nav/_nav.scss

```diff
@@ -54,6 +54,11 @@
         overflow: visible;
       }
     }
+    
+    &--second-level,
+    &--third-level {
+      z-index: 1;
```

Comment:
I kind of like this style of handling z-index: https://css-tricks.com/handling-z-index/